tests/monster_test_suffix.hpp

```diff
@@ -0,0 +1,3594 @@
+// automatically generated by the FlatBuffers compiler, do not modify
```

Comment:
I understand its good to have this test, but now every change to the C++ codegen is going to show 2 copies in the diffs from now on. Maybe we can verify the file exists, then delete it?

Reply:
sounds good, I will do the comparison and delete the file. I'll remove the file from the check in as well.